pages/faq.vue

```diff
@@ -245,7 +245,7 @@ export default {
             question: "How does the Covid Watch app protect my privacy?",
             answer:
               "<p>In short, we don’t collect data about who you are, where you go or who you’re near. With the Covid Watch app, your phone periodically sends out Bluetooth signals, essentially saying to nearby phones, “Hey, I’m here!” If another phone nearby also has the app and says, “Hey, I’m here too,” both phones will log the interaction and store it for two to four weeks right there inside your device.</p>" +
-              "<p>If you are diagnosed with COVID-19, you can choose to share your test results with other Covid Watch users after they are verified by local health officials.  Once you do that, you upload a random number to our database that allows other devices to recognize that the Bluetooth numbers they list are associated with a positive result. The database contains no personal information and no locations. A hacker who broke into our database or your phone would just find a list of random numbers.</p>"
+              "<p>If you are diagnosed with COVID-19, you can choose to share your test results with other Covid Watch users. Once your results have been verified by local health officials, your phone will upload a random number to our database. Other Covid Watch users whose phones have stored the same random number will then be notified that they were exposed to someone who tested positive for COVID-19. Our database contains no personal information and no locations. A hacker who broke into our database or your phone would just find a list of random numbers.</p>"
```

Comment:
(Moving this comment over from #66)

Since it doesn't show up well in the diff, this is a reword from:

> If you are diagnosed with COVID-19, you can choose to share your test results with other Covid Watch users after they are verified by local health officials. Once you do that, you upload...

to
> If you are diagnosed with COVID-19, once your test results are verified by local health officials you can choose to share your results with other Covid Watch users. Once you do that, you upload...

Agree that the first version is clunky, but I also find the second one a bit hard to read. @jcolla-holla, thoughts on:

> If you are diagnosed with COVID-19, you can choose to share your results with other Covid Watch users. Once local health officials verify your results, you upload...

which turns into:
```suggestion
              "<p>If you are diagnosed with COVID-19, you can choose to share your results with other Covid Watch users. Once local health officials verify your results, you upload a random number to our database that allows other devices to recognize that the Bluetooth numbers they have stored are associated with a positive result. The database contains no personal information and no locations. A hacker who broke into our database or your phone would just find a list of random numbers.</p>"
```

Reply:
In fact, nearly that rewording appears in #69 - I'm going to close this PR, as that PR has an (updated) superset of the changes in this PR.